src/test/java/spaceEngineers/SpaceEngEnvironmentTest.java

```diff
@@ -75,6 +75,8 @@ public void observeBlocksTest() {
         Assertions.assertNotNull(observation);
         Assertions.assertNotNull(observation.blocks);
         Assertions.assertTrue(observation.blocks.size() > 0, "Expecting non-zero block count.");
+        Assertions.assertEquals(1, observation.grids.size(), "Expecting 1 grid count.");
+        Assertions.assertEquals(1, observation.grids.get(0).blocks.size(), "Expecting 1 grid block count.");
```

Comment:
I'm running the tests also against the game, where there could be more blocks. We should probably separate tests which are supposed to run against the game and require some setup in the game.